categories/dns.yaml

```diff
@@ -35,6 +35,9 @@ Zones:
 - "Domain Zone Files"
 - "DNS Hosting"
 - "Dynamic DNS"
+- "External DNS Providers"
```

Comment:
Didn't we agree to change the term to Integrated DNS Providers?

Reply:
Yes, we'll be updating the UI and support articles accordingly 👍